cpp/ps0003/cpp_ps0003_solution.cpp

```diff
@@ -2,11 +2,17 @@
 using namespace std;
 int main()
 {
+	// prompting user for number 2/2
 	int x;
-	cout << "enter a namber\n";
+	cout << "enter a number\n";
 	cin >> x;
+

```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number